plugins/omar-oms-plugin/src/main/groovy/omar/oms/JaiImage.groovy

```diff
@@ -43,7 +43,7 @@ class JaiImage
 
     modifiedImage
   }
-  static def reformatImage(def image, int tileWidth = 256, int tileHeight = 256)
+  static def reformatImage(def image /*, int tileWidth = 256, int tileHeight = 256*/)
```

Comment:
Remove block comment in function "/* */". This makes for a confusing looking function signature.

Reply:
Resolved all of the above comments.